_posts/2018-12-03-how-to-make-golden-egg-fried-rice.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "How To Make Golden Fried Rice"
+date: 2018-12-03 21:20:12
+tags:
+- Food
+- chinese cuisine
+categories: food recipes
+twitter_text: "How To Make Golden Fried Rice"
+authors: Cheng Yang
+image: https://c1.staticflickr.com/3/2466/3708994257_4e359233c9_b.jpg
+image_url: https://www.flickr.com/photos/35856236@N06/3708994257/
+image_credit: hawaiianfoodrecipe
+---
+
+Egg Fried Rice is a very homely dish in China. Every Chinese family make their fried rice differently, so you don't actually need a receipt to make fried rice, you can add whatever you like into the rice. Today, I am going to show you my method to make this simple, quick and delicious dish.
+
+## Ingredients
+- 400g cooked rice (For two) - I use `Nishiki Medium Grain Rice` (https://www.amazon.com/Nishiki-Medium-Grain-Rice-Pound/dp/B00852ZN2U/ref=sr_1_8_s_it?s=grocery&ie=UTF8&qid=1543877113&sr=1-8&keywords=rice)
+- 3 eggs
+- half onion (Chopped)
+- 80 gram luncheon meat / any other meats
+- corns and green peas
+- 1/2 tbp salt
+- 1/2 tbp sugar
+- 3/4 tbp white pepper powder
+- 4 Thai Pepper (Optional)
+- appropriate amount of scallion
```

Comment:
`appropriate amount of scallion` is a little vague. Maybe `scallion for garnish`?